src/main.ts

```diff
@@ -3,6 +3,6 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  await app.listen(3000);
+  await app.listen(3010);
```

Comment:
PullProofAI

The port number has been changed in the `main.ts` file. Ensure this change aligns with the overall configuration and deployment setup.
```diff
-  await app.listen(3000);
+  await app.listen(3010);
```


<!-- This is an auto-generated comment by PullProofAI -->